scripts/Interface/ui/drill/model/DrillExportModel.py

```diff
@@ -178,34 +179,50 @@ def _logSuccessExport(self, wsName):
 
     def run(self, sample):
         """
-        Run the export algorithms on a workspace. If the provided workspace is
-        a groupworkspace, the export algorithms will be run on each member of
-        the group.
+        Run the export algorithms on a sample. For each export algorithm, the
+        function will try to validate the criteria (using _validCriteria()) on
+        the output workspace that corresponds to the sample. If the criteria are
+        valid, the export will be run on all workspaces whose name contains the
+        sample name.
 
         Args:
-            workspaceName (str): name of the workspace
+            sample (DrillSample): sample to be exported
         """
         exportPath = config.getString("defaultsave.directory")
+        if not exportPath:
+            exportPath = os.getcwd()
         workspaceName = sample.getOutputName()
 
+        try:
+            outputWsGroup = mtd[workspaceName]
```

Comment:
I just realized that output workspace doesn't have to be a workspace group. I think we should check that too.

Reply:
I added a check for that.